tools/flash_algo/flash_algo.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/env python
```

Comment:
Discussed in person. Summary - long term this file should be moved into arm-pack-manager or another common repo. Right now the master copy lives in the [FlashAlgo](https://github.com/mbedmicro/FlashAlgo/blob/master/scripts/flash_algo.py) repository and this is a copy of it.